scripts/koji/koji_build.py

```diff
@@ -50,6 +50,30 @@ def koji_url(remote, hash):
         raise Exception("Unrecognized remote URL")
     return remote + "?#" + hash
 
+@contextmanager
+def local_branch(branch):
+    prev_branch = subprocess.check_output(['git', 'branch', '--show-current']).strip()
+    subprocess.check_call(['git', 'checkout', '-b', branch])
+    try:
+        yield branch
+    finally:
+        subprocess.check_call(['git', 'checkout', prev_branch])
+        subprocess.check_call(['git', 'branch', '-D', branch])
+        subprocess.check_call(['git', 'push', '--delete', 'origin', branch])
+
+@contextmanager
+def push_bumped_release(git_repo, trigram):
+    uuid = uuid4()
+    branch = f'koji/test/{uuid}'
```

Comment:
maybe use the trigram in branch name too?

Reply:
I did that latter in the commits

Reply:
Yes, but in a commit that already does 2 other things (switch from uuid to date, and branch cleanup).  It would be better to fold the switch to `f'koji/test/{trigram}/{t}'` completely in the original commit, so `uuid` also never gets introduced.  That way this other commit will have the single purpose of branch cleanup.